### ISSUE TITLE
[post 1.7] Allow reloading addon files

### ISSUE THREAD

**Issue by SevenSpheres** [CelestiaProject/Celestia#531]
Note: this is probably also something for after the 1.7 release.

It should be possible to "reload" Celestia addon files while Celestia is running. Then addon developers (such as myself) wouldn't need to restart Celestia after making changes to an addon file.

**Comment by Panterstruck** [comment]
Yes this is desperately needed. You cant imagine how many hours i have wasted restarting celestia to check my addon changes.

This goes a step further but allowing to change certain parameters you would define in ssc files in celestia itself would help me save so many headaches. Reloading addon files is a more straight forward way to archieve this tho i guess.

**Comment by Anthony-B-Russo10** [comment]
Same goes for me, this is needed.

**Comment by 375gnu** [comment]
In 1.7 you can use load_fragment CELX function

**Comment by Anthony-B-Russo10** [comment]
Ah

**Comment by SevenSpheres** [comment]
> In 1.7 you can use load_fragment CELX function

What I'm asking for is the ability to edit an .ssc, .stc, or .dsc file, save it, and then re-load it into Celestia without restarting Celestia. (but that's for after 1.7)

**Comment by Anthony-B-Russo10** [comment]
Well 1.7.0 at least.
